README.md

```diff
@@ -61,6 +61,26 @@ Press CTRL+C to quit
 
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+## Running with Docker
+
+To run the app in a docker container, run from the root directory of the project:
+
+```bash
+$ docker build --tag todo-app:dev .
```

Comment:
```suggestion
$ docker build --tag todo-app:dev --target development .
```
I suspect this was working before, but since the target isn't specified it defaults to the final stage in the Dockerfile, which is now your testing stage!